technologies/BehaviorTree.CPP/README.md

```diff
@@ -2,10 +2,12 @@
 
 This package provides some ready to use Nodes (i.e. "actions") to be used with **pyrobosim**.
 
-We suggest creating your XML trees in the folder `technologies/BehaviorTree.CPP/pyrobosim_btcpp/trees`
+We suggest creating your XML trees in the folder `/delib_ws/src/technologies/BehaviorTree.CPP/pyrobosim_btcpp/trees`
```

Comment:
I think it's fine to leave this path as is, relative to the repo. Since people can also develop on the host.

Reply:
ok